backup/backup.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+#
+# Back up an instance and store the data in the local directory.
+#
+# NOTE that to prevent inconsistent data, the MASTODON SERVICE WILL BE STOPPED for the duration of the backup.
+#
+# Backup can run either locally or remote. To run locally, start this script without options.
+#
+# For remote mode, which uses scp/ssh, supply "user@remote_host" and a private key file as first and second argument.
+
+set -euo pipefail
+backup_basedir="/opt/backup"
+
+remote=""
```

Comment:
I'd recommend you to actually use an array, especially that the `remote` variable will contain user-passed paths:
```suggestion
remote=()
```